conversions/hex-bin.py

```diff
@@ -0,0 +1,58 @@
+#  CONVERT HEXADECIMAL TO BINARY
+import math
+from typing import Union
+
+
+def convert(num: str) -> Union[bool, int]:
```

Comment:
I know that this was recommended in a previous review but I would like to request a change.

When someone calls this function, they expect to get back a binary number which they may then use in a mathematical calculation.  Returning a bool or an int in this case is not considered Pythonic because it breaks their expectations so let's agree that this function should return an int or should raise an Exception if garbage data is provided as input.  This is the way that Python's builtin functions behave:
```
>>> hex(123)
'0x7b'
>>> hex("dog")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'str' object cannot be interpreted as an integer
```

Also, the name `convert()` is not very self-documenting and neither is `num` so let's go for:
```suggestion
def hex_to_bin(hex_num: str) -> int:
```

Reply:
Apologies to the contributor for derailing but this makes more sense.